src/test/scala/nl/knaw/dans/easy/stage/EasyStageDatasetSpec.scala

```diff
@@ -100,19 +100,20 @@ class EasyStageDatasetSpec extends FlatSpec with Matchers {
 
     createProps()
 
-    // license-by-url seems to require mocking web-access, probably beyond the purpose of this test
-    def useTestBag(f: File) = f.getName != "additional-license-by-url"
+    // license-by-url seems to require mocking web-access, probably beyond the purpose of this test,
+    // av is tested seperately
+    def useTestBag(f: File) = f.getName != "additional-license-by-url" && f.getName != "minimal-av"
```

Comment:
Maybe other names get added here later, so before expanding this expression even further you could make this into the following:

```scala
List("additional-license-by-url", "minimal-av").forall(_ != f.getName)
```

That way, future names can just be added to the list.